### PR TITLE
More README content

### PR DESCRIPTION
**Pull request by jamiehannaford**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,40 @@
...
+
+If you are a Linux or OSX user, you will need to download the binary like so:
+
+```bash
+wget https://github.com/blah
+```
+
+If you are a Windows user, you will need to enter the following link in your browser:
+
+> https://github.com/blah
```
Comment (dstufft) [inline README.md]: Not sure if we want to mention it, but if they have powershell they can also do:

```
Invoke-WebRequest https://github.com/blah -OutFile cs-reboot-info
```

Or something along those lines. Don't know if it's worth mentioning though.

Comment (DonSchenck) [inline README.md]: gophercloud MUST be installed as well.

Comment (smashwilson) [inline README.md]: Go binaries are self-contained. They'll only need Gophercloud manually if they're building from source.

**Comment by smashwilson** [comment]
Superseded by #6.

**Pull request closed without merging**